offerzen/global/employers/prospects@1.0.js

```diff
@@ -0,0 +1,177 @@
+window.$loaded(function () {
+  window.$parsleyLoaded = function (cb) {
+    setTimeout(() => {
+      if (window.Parsley && window.ParsleyValidator) {
+        cb(window, document, parsley, undefined);
+        return;
+      }
+      $parsleyLoaded(cb);
+    }, 50);
+  };
+  window.$parsleyLoaded(function (window, document, parsley) {
+    function trackSubmission() {
+      var event = $('.js-analytics-event').text();
+      var action = $('.js-analytics-action').text();
+      var label = $('.js-analytics-label').text();
+      var category = $('.js-analytics-category').text();
+      var source = $('.js-analytics-source').text();
+      // dataLayer.push({
```

Comment:
If this isn't needed, let's remove it

Reply:
Remove dataLayer.push, confirmed with B not needed